dataRetriever/resolvers/trieNodeResolver.go

```diff
@@ -208,6 +218,40 @@ func (tnRes *TrieNodeResolver) sendResponse(serializedNodes [][]byte, message p2
 	return tnRes.Send(buff, message.Peer())
 }
 
+func (tnRes *TrieNodeResolver) sendLargeMessage(
+	largeBuff []byte,
+	reference []byte,
+	chunkIndex int,
+	message p2p.MessageP2P,
+) error {
+
+	log.Warn("assembling chunk", "reference", reference, "len", len(largeBuff))
```

Comment:
is the log.Warn needed here? Is log.Debug not enough?

Reply:
added TODO to remove when the feature is complete

Reply:
ok